# Problem 1
### have a list of numbers starting from 1 to 100.have a second list containing numbers starting from 100 to 200.<br>merge two list and make a combined list. 
### filter out odd and even numbers from combined list and create a odd_numbers list and even_numbers list separately

In [1]:
firstl = []
secondl = []
for i in range(1,101):
    firstl.append(i)
for i in range(101,201):
    secondl.append(i)
firstl.extend(secondl)
print(firstl)

oddl = []
evenl = []
for i in firstl:
    if i%2 == 0:
        evenl.append(i)
    else:
        oddl.append(i)

print(oddl)
print(evenl)

[1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190, 191, 192, 193, 194, 195, 196, 197, 198, 199, 200]
[1, 3, 5, 7, 9, 11, 13, 15, 17, 19, 21, 23, 25, 27, 29, 31, 33, 35, 37, 39, 41, 43, 45, 47, 49, 51, 53, 55,

# Problem 2
### let a list have all the first 10 alphabetic characters as 10 elements. randomly select any one character from list using random method we covered on day 1. <br>if the randomly selected character is a 'z' , then, print the message 'z is picked from the set' else print 'no,its not picked' 

In [2]:
import random

alph_list = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j']
character = random.choice(alph_list)
if character == 'z':
    print('z is picked from the set')
else:
    print('no, z is not picked')

no, z is not picked


# Problem 3
### get your empid and name as input from console. e:g: 101 Python or 102 Java - your empid and name should be separated by a space.<br>everytime you get the input,do the below validations. check if the input string has a space, if so, split the string and assign to empid and name variables.<br>the empid should not not be more than 3 chars and name should not exceed 10 chars. if not,print appropriate messages. 
### remove the leading and trialing space in both name and empid string. 
### get the position of character 'z' in name string . if not found, concatenate your string with a 'z' and print the concatenated name. 

In [3]:
user_inp = input('Enter empid and name: ').split()
empid = user_inp[0]
name = user_inp[1]
if len(empid) > 3:
    print('Wrong empid')
elif len(name) > 10:
    print('Name too long')
empid.strip()
name.strip()

if name.find('z')  == -1:
    name = name + 'z'
    print(name)
else:
    print(name.find('z'))

Enter empid and name: 101 dave
davez
